ex03.ex

```diff
@@ -149,8 +219,74 @@ defmodule Ex03 do
   Think a little about a nice way to lay this code out.
   """
 
-  def won . . . "your code"
+    def won(state) do
```

Comment:
# -4

This can all be done using a simple pattern match:

~~~ elixir
  def won1({ a, a, a,  _, _, _,  _, _, _ }), do: a
  def won1({ _, _, _,  a, a, a,  _, _, _ }), do: a
  def won1({ _, _, _,  _, _, _,  a, a, a }), do: a
  def won1({ a, _, _,  a, _, _,  a, _, _ }), do: a
  def won1({ _, a, _,  _, a, _,  _, a, _ }), do: a
  def won1({ _, _, a,  _, _, a,  _, _, a }), do: a
  def won1({ a, _, _,  _, a, _,  _, _, a }), do: a
  def won1({ _, _, a,  _, a, _,  a, _, _ }), do: a
  def won1(_), do: false
~~~